app/models/item.py

```diff
@@ -35,4 +35,6 @@ def to_dict(self):
             'createdAt': self.created_at,
             'updatedAt': self.updated_at,
             'seller': self.seller.to_dict(),
+            'category': self.category.name,
```

Comment:
Since this isn't related to reviews, let's remove this. I believe @isaacsungpak is adding the category.

Reply:
got it, thanks.